File: SWIFT/hellodeath!.swift
import Foundation

typealias H = String
typealias E = String.Element

func add(_ string: H, _ element: E) -> H {
    return string + String(element)
}

infix operator ⚡️: AdditionPrecedence

func ⚡️(_ string1: H, _ string2: H) -> H {
    return string1 + string2
}

let h: H = "H"
let e: E = "e"
let l: E = "l"
let o: E = "o"
let space: E = " "
let w: E = "W"
let r: E = "r"
let d: E = "d"
let exclamation: E = "!"

let helloWorld: H = (h ⚡️ add("", e) ⚡️ add("", l) ⚡️ add("", l) ⚡️ add("", o)) ⚡️ add("", space) ⚡️ add("", w) ⚡️ add("", o) ⚡️ add("", r) ⚡️ add("", l) ⚡️ add("", d) ⚡️ String(exclamation)

print(helloWorld)

File: PASCAL/hellodeath!.pas
program HelloWorld(output);

const
  H = 'H';
  e = 'e';
  l = 'l';
  o = 'o';
  W = 'W';
  r = 'r';
  d = 'd';
  space = ' ';
  exclamation = '!';

var
  Hello: string[5];
  World: string[5];

begin
  Hello := H + e + l + l + o;
  World := W + o + r + l + d;

  writeln(Hello, space, World, exclamation);
end.